orchestrator/src/config.rs

```diff
@@ -0,0 +1,156 @@
+    pub start_delay_seconds: u64,
+    #[serde(default = "default_config")]
+    pub config: HotShotConfigFile,
+    #[serde(default = "default_web_server_config")]
+    pub centralized_web_server_config: Option<CentralizedWebServerConfig>,
+}
+
+fn default_web_server_config() -> Option<CentralizedWebServerConfig> {
+    None
+}
+
+impl<K, E> From<NetworkConfigFile> for NetworkConfig<K, E> {
+    fn from(val: NetworkConfigFile) -> Self {
+        NetworkConfig {
+            rounds: val.rounds,
+            transactions_per_round: val.transactions_per_round,
+            node_index: 0,
+            seed: val.seed,
+            padding: val.padding,
+            config: val.config.into(),
+            key_type_name: std::any::type_name::<K>().to_string(),
+            election_config_type_name: std::any::type_name::<E>().to_string(),
+            start_delay_seconds: val.start_delay_seconds,
+            centralized_web_server_config: val.centralized_web_server_config,
+        }
+    }
+}
+
+/// Holds configuration for a `HotShot`
+#[derive(Debug, Clone, serde::Serialize, serde::Deserialize)]
+pub struct HotShotConfigFile {
+    /// Total number of nodes in the network
+    pub total_nodes: NonZeroUsize,
+    /// Maximum transactions per block
+    pub max_transactions: NonZeroUsize,
+    /// Minimum transactions per block
+    pub min_transactions: usize,
+    /// Base duration for next-view timeout, in milliseconds
+    pub next_view_timeout: u64,
+    /// The exponential backoff ration for the next-view timeout
+    pub timeout_ratio: (u64, u64),
+    /// The delay a leader inserts before starting pre-commit, in milliseconds
+    pub round_start_delay: u64,
+    /// Delay after init before starting consensus, in milliseconds
+    pub start_delay: u64,
+    /// Number of network bootstrap nodes
+    pub num_bootstrap: usize,
+    /// The minimum amount of time a leader has to wait to start a round
+    pub propose_min_round_time: Duration,
+    /// The maximum amount of time a leader can wait to start a round
+    pub propose_max_round_time: Duration,
+}
+
+impl<K, E> From<HotShotConfigFile> for HotShotConfig<K, E> {
+    fn from(val: HotShotConfigFile) -> Self {
+        HotShotConfig {
+            execution_type: ExecutionType::Continuous,
+            total_nodes: val.total_nodes,
+            max_transactions: val.max_transactions,
+            min_transactions: val.min_transactions,
+            known_nodes: Vec::new(),
+            next_view_timeout: val.next_view_timeout,
+            timeout_ratio: val.timeout_ratio,
+            round_start_delay: val.round_start_delay,
+            start_delay: val.start_delay,
+            num_bootstrap: val.num_bootstrap,
+            propose_min_round_time: val.propose_min_round_time,
+            propose_max_round_time: val.propose_max_round_time,
+            election_config: None,
+        }
+    }
+}
+
+// This is hacky, blame serde for not having something like `default_value = "10"`
```

Comment:
I thought we could just add `#[serde(default = "10")]` on the `rounds` field. 😮

Reply:
This is actually a comment Victor left on the old orchestrator that I copied over.  But I tried what you suggested, and it doesn't seem to work. 😞